subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -157,7 +157,7 @@ include::sample[dir="userguide/tasks/configureUsingBlock/groovy",files="build.gr
 include::sample[dir="userguide/tasks/configureUsingBlock/kotlin",files="build.gradle.kts[tags=configure]"]
 ====
 
-This works for _any_ task. Task access is just a shortcut for the `tasks.getByName()` method. It is important to note that if you pass a block to the `getByName()` method, this block is applied to _configure_ the task, not when the task executes.
+This works for _any_ task. Task access is just a shortcut for the `tasks.named()` method. It is important to note that if you pass a block to the `named()` method, this block is applied to _configure_ the task, not when the task executes.
```

Comment:
❌ I don't think this change is correct and we should expand on this a bit more. In Groovy, the shortcut is just like `tasks.getByName()`, however, in Kotlin the shortcut is just like `tasks.named()`. Due to the change we did to the sample, the [`configureUsingBlock` sample](https://github.com/gradle/gradle/blob/master/subprojects/docs/src/samples/userguide/tasks/configureUsingBlock/groovy/build.gradle) doesn't make any sense and should probably be reverted.